internal/scaffold/role.go

```diff
@@ -148,13 +148,9 @@ func updateRoleFile(role interface{}, roleFilePath string) error {
 	if err != nil {
 		return fmt.Errorf("failed to marshal role(%+v): %v", role, err)
 	}
-	m := &map[string]interface{}{}
-	if err := yaml.Unmarshal(d, m); err != nil {
-		return fmt.Errorf("failed to unmarshal role(%+v): %v", role, err)
-	}
-	data, err := yaml.Marshal(m)
+	data, err := yaml.JSONToYAML(d)
```

Comment:
We can get rid of the above `json.Marshal` call and this `yaml.JSONToYAML` call and replace them with `yaml.Marshal` [since that's actually what this YAML library is doing under the hood](https://github.com/kubernetes-sigs/yaml/blob/9fc95527decd95bb9d28cc2eab08179b2d0f6971/yaml.go#L16-L28).

Reply:
|o| sorry for that. Done.